contracts/access/manager/IManaged.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+interface IManaged {
```

Comment:
Why not keeping the `AccessManagedUnauthorized` error and the `AuthorityUpdate` event in this interface?

Reply:
moved